app/src/test/java/ru/krivocraft/kbmp/utils/SecondsTest.java

```diff
@@ -0,0 +1,43 @@
+package ru.krivocraft.kbmp.utils;
```

Comment:
add license header

Reply:
can we leave the whole license header work to issue #23 please?